src/mem/mem_ctrl.hh

```diff
@@ -81,7 +80,8 @@ class BurstHelper
 
     BurstHelper(unsigned int _burstCount)
         : burstCount(_burstCount), burstsServiced(0)
-    { }
+    {
```

Comment:
unneeded change. I won't mark all of these, but please look at the diff on github or in VS code and see what's not important and revert those parts.